README.md

```diff
@@ -69,7 +69,7 @@ Prerequisites for building Flink:
 
 * Unix-like environment (we use Linux, Mac OS X, Cygwin, WSL)
 * Git
-* Maven (we recommend version 3.8.6 and require at least 3.1.1)
+* Maven (we require at least version 3.8.6)
```

Comment:
10 lines below there is a note about requirements of 3.1.1 possibility to build with 3.3.x, shouldn't it be updated as well?